Resfeber/Model Controller/DestinationController.swift

```diff
@@ -9,62 +9,91 @@
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "Lisbon",
-                                                            image: UIImage(named: "Lisbon")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "CozumelTwo",
-                                                            image: UIImage(named: "Cozumel")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "SeattleTwo",
-                                                            image: UIImage(named: "Seattle")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "PhiladelphiaTwo",
-                                                            image: UIImage(named: "Philidelphia")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "BaliTwo",
-                                                            image: UIImage(named: "Bali")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "LisbonTwo",
-                                                            image: UIImage(named: "Lisbon")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "CozumelThree",
-                                                            image: UIImage(named: "Cozumel")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "SeattleThree",
-                                                            image: UIImage(named: "Seattle")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "PhiladelphiaThree",
-                                                            image: UIImage(named: "Philidelphia")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "BaliThree",
-                                                            image: UIImage(named: "Bali")!,
-                                                            isFavorite: false))
-            DestinationData.destinations.append(Destination(name: "LisbonThree",
-                                                            image: UIImage(named: "Lisbon")!,
-                                                            isFavorite: false))
+    func readDestinations() {
```

Comment:
Not sure I get the point of appending all of these destinations. If `searchDestinations.count == 0` then why not just set `searchDestinations = [Destination(name:...), Destination(name:..), ...]`

Reply:
You're right, assignment during initialization makes more sense. We will be removing this mock data all together in the next week or so.